test/livebook/runtime/evaluator_test.exs

```diff
@@ -1300,6 +1300,69 @@ defmodule Livebook.Runtime.EvaluatorTest do
       assert_receive {:runtime_evaluation_response, :code_1, terminal_text("6"), metadata()}
     end
 
+    test "evaluate erlang-module code", %{evaluator: evaluator} do
+      Evaluator.evaluate_code(
+        evaluator,
+        :erlang,
+        "-module(tryme). -export([go/0]). go() ->{ok,went}.",
+        :code_4,
+        []
+      )
+
+      assert_receive {:runtime_evaluation_response, :code_4, terminal_text(_), metadata()}
+    end
+
+    test "evaluate erlang-module error function already defined", %{evaluator: evaluator} do
+      Evaluator.evaluate_code(
+        evaluator,
+        :erlang,
+        "-module(tryme). -export([go/0]). go() ->{ok,went}. go() ->{ok,went}.",
+        :code_4,
+        []
+      )
+
+      assert_receive {
+        :runtime_evaluation_response,
+        :code_4,
+        error(_),
```

Comment:
Since we are testing specific errors, we should make some assertions on the error content to make sure it's as expected.

Reply:
Is there a error_match("some partial-string must be present") which I missed? That would be handy since you could test for a partial string - without having to copy the entire error string , which will be full or additional meta-info such as line-numbers etc.

Reply:
You can assign the message and then assert separately, like this:

https://github.com/livebook-dev/livebook/blob/a7fe8f6660e9e641130080d276e1185ea69389e5/test/livebook/runtime/evaluator_test.exs#L220-L239

or this:

https://github.com/livebook-dev/livebook/blob/a7fe8f6660e9e641130080d276e1185ea69389e5/test/livebook/runtime/evaluator_test.exs#L1434-L1436